cmd/cli/install.go

```diff
@@ -131,7 +131,18 @@ func (i *installCmd) run(config *helm.Configuration) error {
 	installClient.Wait = true
 	installClient.Atomic = i.atomic
 	installClient.Timeout = i.timeout
+
+	debug("Beginning OSM installation")
 	if _, err = installClient.Run(i.chartRequested, values); err != nil {
+		if !settings.Verbose {
```

Comment:
For consistency, it's better to implement `func (s *EnvSettings) Verbose() { return s.verbose }` in `pkg/cli`, and use `settings.Versobse()` here.